py_gtfs_rt_ingestion/ingest.py

```diff
@@ -72,10 +84,9 @@ def main(files: list[str]) -> None:
     config = Configuration(filename=files[0])
 
     try:
-        INGEST_BUCKET = os.environ['ingest_bucket']
-        OUTPUT_BUCKET = os.environ['OUTPUT_DIR']
-        ARCHIVE_BUCKET = os.environ['archive_bucket']
-        ERROR_BUCKET = os.environ['error_bucket']
+        EXPORT_BUCKET = os.join(os.environ['EXPORT_BUCKET'], DEFAULT_S3_PREFIX)
+        ARCHIVE_BUCKET = os.join(os.environ['ARCHIVE_BUCKET'], DEFAULT_S3_PREFIX)
+        ERROR_BUCKET = os.join(os.environ['ERROR_BUCKET'], DEFAULT_S3_PREFIX)
```

Comment:
use the correct buckets from the lambda environment that were set while parsing args.

i joined the buckets with the expected prefix. i'm assuming pyarrow.fs.s3filesystem is going to be able to handle this as expected.